src/queries.test.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { ReactNode } from "react";
+import {
+  onlineManager,
+  QueryClient,
+  QueryClientProvider,
+} from "@tanstack/react-query";
+import {act, renderHook, waitFor} from "@testing-library/react";
+import {addDefaultMutationFns, useSubmission, useSubmissionList} from "./queries";
+import {produce} from "immer";
+import {patchMetadataSubmissionError, server, delay} from "./mocks/server";
+
+interface TestWrapperProps {
+  children: ReactNode
+}
+const createWrapper = () => {
+  const queryClient = new QueryClient({
```

Comment:
The docs for `QueryClient` are here: https://tanstack.com/query/latest/docs/reference/QueryClient